src/neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -86,7 +86,7 @@ internal protected virtual void Mint(ApplicationEngine engine, UInt160 account,
             BigInteger totalSupply = new BigInteger(storage.Value);
             totalSupply += amount;
             storage.Value = totalSupply.ToByteArrayStandard();
-            engine.SendNotification(Hash, new Array(new StackItem[] { "Transfer", StackItem.Null, account.ToArray(), amount }));
+            engine.SendNotification(Hash, new Array(new StackItem[] { actionName, StackItem.Null, account.ToArray(), amount }));
```

Comment:
We need both, that's the reason for add a new one parameter in NEP5, after amount, we could send an optional value for the reason.

Reply:
Ok，I add an optional value for reason after amount.

Reply:
It's only my idea, but we should modify the NEP5

Reply:
Yes, we should modify the NEP5 as well. @erikzhang How do you think about this?

Reply:
What do you think about this @erikzhang ?

Reply:
Add what parameter?

Reply:
@shargon  If we don't use payback, we don't have to add the `reason` field in `transfer` event in NEP-5.
```
public static event transfer(byte[] from, byte[] to, BigInteger amount, byte[] reason)
```